ui/app/utils/wait.js

```diff
@@ -0,0 +1,10 @@
+import RSVP from 'rsvp';
```

Comment:
Seems weird that the fn is `timeout` which is also the default export from a file called `wait`. Not a big deal, but I'd rename the file.

Reply:
Good catch. I just save as'd the existing `timeout.js`. I'll rename the function here.